### ISSUE TITLE
Bucket of Tadpale cannot be sold easily

### ISSUE THREAD

**Issue by JustEli** [ChestShop-authors/ChestShop-3#519]
### What is happening?

Bucket of Tadpoles store NBT data called "Age". Age is in ticks, and because of this, it is impossible to have more than one bucket in stock for a chest shop.

ChestShop creates a different ID for every Bucket of Tadpole:
![image](https://user-images.githubusercontent.com/8517465/173438611-8f2fae7f-ccf6-45d9-bb76-41471315208a.png)

Showing the NBT data of the three different Bucket of Tadpoles:
![image](https://user-images.githubusercontent.com/8517465/173438680-b381793a-679a-4b85-9aa2-426d69e63666.png)

### What did you expect to happen?

Bucket of Tadpole to be all the same ID

### Plugin Version

3.12.1 build 347

### Plugin Config

-

### Server Version

git-Purpur-1659 (MC: 1.19)

### Server Log

-

### What other plugins are you running?

None, as it was on a test server

### Agreements

- [X] I have checked if there are already similar issues open.
- [X] I am running the latest development version available from https://ci.minebench.de/job/ChestShop-3/.
- [X] The version [has support](https://github.com/ChestShop-authors/ChestShop-3/security/policy#supported-versions) for this type of issue.
- [X] I have provided all requested information and agree to the terms of the [project's license](https://github.com/ChestShop-authors/ChestShop-3/blob/master/LICENSE)

**Comment by Phoenix616** [comment]
Not sure this is a bug tbh. A player might care what kind of extra values like the Tadpoal age is on the item when purchasing it. If anything this needs to be included in the messages on /iteminfo and /shopinfo and the eventual wildcard support (#421) be used for selling all kinds of buckets.

**Comment by JustEli** [comment]
definitely a bug to me: there is absolutely no way to sell multiple tadpole buckets per chest shop, just one, because the age is in ticks

**Comment by ryankayz7** [comment]
I agree with @JustEli. My players are also having issues with this. It forces players to literally make a shop for every tadpole as every tadpole has a different ID when doing the /iteminfo command. Time is fair to mention but if that is something people want they shouldn't have to make a new chestshop for every tadpole they want to sell
